static/_redirects

```diff
@@ -91,7 +91,8 @@
 /docs/concepts/cluster-administration/guaranteed-scheduling-critical-addon-pods/     /docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/ 301
 /docs/concepts/cluster-administration/master-node-communication/     /docs/concepts/architecture/master-node-communication/ 301
 /docs/concepts/cluster-administration/network-plugins/     /docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/ 301
-/docs/concepts/cluster-administration/out-of-resource/     /docs/tasks/administer-cluster/out-of-resource/ 301
+/docs/concepts/cluster-administration/out-of-resource/     /docs/concepts/scheduling-eviction/pod-eviction/ 301   
+/docs/tasks/administer-cluster/out-of-resource/     /docs/concepts/scheduling-eviction/pod-eviction/ 301
```

Comment:
This redirect should ideally be grouped with other tasks-section redirects.

Reply:
Ah, sorry, I was grouping the actual topics together. I'll move it.